_browser/blockstack_storage.md

```diff
@@ -578,8 +532,13 @@ process URL paths that contain the `.` (dot) character for example,
 3. Add a single method to the `Profile` class that determines if the app is viewing the local user's profile or another user's profile.
```

Comment:
The ` When you are done, the import statement should look like the following:` is in code style. Should remove the tabs